Yoast/wordpress-seo
# Change the namespace

## Summary


This PR can be summarized in the following changelog entry:

* N/A - No technical change.

## Test instructions

This PR can be tested by following these steps:

* Make sure everything still works and verify that Travis is happy 😄 

Please merge this pull into the indexables branch.
